pkg/apis/trustmanager/v1alpha2/types_cluster_bundle.go

```diff
@@ -212,13 +212,52 @@ type SourceObjectKeySelector struct {
 	IncludeAllKeys bool `json:"includeAllKeys,omitempty"`
 }
 
+// TargetTemplate defines the form of the Kubernetes Secret or ConfigMap bundle targets.
+type TargetTemplate struct {
+	// Key is the key of the entry in the object's `data` field to be used.
+	// +kubebuilder:validation:MinLength=1
+	Key string `json:"key"`
+
+	// Metadata is an optional set of labels and annotations to be copied to the target.
+	// +optional
+	Metadata *TargetMetadata `json:"metadata,omitempty"`
+}
+
+// GetAnnotations returns the annotations to be copied to the target or an empty map if there are no annotations.
+func (t *TargetTemplate) GetAnnotations() map[string]string {
+	if t == nil || t.Metadata == nil || t.Metadata.Annotations == nil {
```

Comment:
```suggestion
	if t == nil || t.Metadata == nil {
```

Another nit, sorry for not spotting this last time.